debugger-launchjson.md

```diff
@@ -18,12 +18,16 @@ The `preLaunchTask` field runs the associated taskName in tasks.json before debu
 This will create a task that runs `dotnet build`. You can read more about tasks at [https://code.visualstudio.com/docs/editor/tasks](https://code.visualstudio.com/docs/editor/tasks).
 
 ## Program
-The program field is set to the path to the application dll or .NET Core host executable to launch. 
+The program field is set to the path of the application dll or .NET Core host executable to launch.
 
-Example: "${workspaceRoot}/bin/Debug/\<target-framework\>/\<project-name.dll\>" where:
+This property normally takes the form: "${workspaceRoot}/bin/Debug/\<target-framework\>/\<project-name.dll\>".
```

Comment:
Why not write `\<project-name\>.dll` (which would also slightly simplify the description)? Are there any cases in which the dll gets placed into the framework folder but with a different name?